src/99-appendices/04-entity-table.md

```diff
@@ -10,23 +10,24 @@ file type are denoted. Entity formats indicate whether the value is alphanumeric
-| headshape<br>(meg)                                                                             | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |
-| markers<br>(meg)                                                                               | REQUIRED      | OPTIONAL      | OPTIONAL       | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |
-| photo<br>(meg/eeg/ieeg)                                                                        | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       |                 |        
-| electrodes<br>(eeg/ieeg)                                                                       | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |
-| events<br>(meg/eeg/ieeg)                                                                       | REQUIRED      | OPTIONAL      | REQUIRED       |               |                          |                |                          | OPTIONAL      |                        |                |                     |                       |                 |        
+| Entity                                                                                         | Subject       | Session       | Task           | Acquisition   | Contrast Enhancing Agent | Reconstruction | Phase-Encoding Direction | Run           | Corresponding modality | Echo           | Recording           | Part               | Processed (on device) | Space           |
+| :--------------------------------------------------------------------------------------------- | :------------ | :------------ | :------------- | :------------ | :----------------------- | :------------- | :----------------------- | :------------ | :--------------------- | :------------- | :------------------ | :----------------- | :-------------------- | :---------------|
+| Format                                                                                         | `sub-<label>` | `ses-<label>` | `task-<label>` | `acq-<label>` | `ce-<label>`             | `rec-<label>`  | `dir-<label>`            | `run-<index>` | `mod-<label>`          | `echo-<index>` | `recording-<label>` | `part-<mag|phase>` | `proc-<label>`        | `space-<label>` |
```

Comment:
here should be no `<mag|phase>` should be just `<label>`.  Then specific modalities would come up with specific labels.

Reply:
Done!

Reply:
changes to table header widths need to propagate, otherwise leads to travis to puke on linting:
```
src/99-appendices/04-entity-table.md
  15:331-15:332  warning  Misaligned table fence  table-pipe-alignment  remark-lint
  15:355-15:356  warning  Misaligned table fence  table-pipe-alignment  remark-lint
  15:373-15:374  warning  Misaligned table fence  table-pipe-alignment  remark-lint
src/99-appendices/05-units.md: no issues found
```

Reply:
Ah, whoops. Thanks for catching that. Should be fixed now.